lib/config/config-file.js

```diff
@@ -19,7 +19,8 @@ var debug = require("debug"),
     ConfigOps = require("./config-ops"),
     validator = require("./config-validator"),
     Plugins = require("./plugins"),
-    resolveModule = require("resolve"),
+    Module = require("module"),
```

Comment:
I can't really find any documentation for `module`, what exactly does it do (or rather I can see that implements Node's module system, but why do you need it here?). It also looks like you are not using it anywhere.